templates/app/app.hbs

```diff
@@ -26,13 +26,20 @@ if (process.env.CORS_ORIGINS) {
   allowedOrigins = allowedOrigins.concat(process.env.CORS_ORIGINS.split(','));
 }
 
-app.use(cors({
+const corsConfig = {
   origin: allowedOrigins,
   allowedHeaders: ['Authorization', 'X-Requested-With', 'Content-Type'],
   maxAge: 86400, // NOTICE: 1 day
   credentials: true,
+};
+
+app.use('/forest/authentication', cors({
+  ...corsConfig,
+  origin: corsConfig.origin.concat('null')
```

Comment:
Looks great to me if it fits your technical needs for the new authentication system implementation.

One remaining questions, I have in mind is: as this generated code will be in the hands of developers, shouldn't we add a comment about this "strange" `'null'` origin? Maybe to say that it is necessary for the OpenId authentication protocol. It could prevent some developers to get rid of it, as the code looks a bit complex for no obvious reason.

Reply:
Ok, I'm adding a comment